system_tasks.yml

```diff
@@ -15,6 +15,26 @@
     - python-dev
     - gdal-bin
     - unzip
+    - python-apt
+
+# By default, Piecewise uses EFF's LetsEncrypt CA for obtaining SSL certificates.
+# If you have obtained SSL certificates from another source, you should comment out the 
+# lines below, ending at the line starting with: `shell: "certbot ...` and then edit 
+# the Nginx default site to point at your certificates: /conf/etc/nginx/sites-available/default
+- apt_repository:
+    repo: deb http://ftp.debian.org/debian jessie-backports main
+    state: present
+- name: Update apt database to get jessie-backports
```

Comment:
Actually, looking at the code now, I see why I made a second apt update and install section. Adding the backports repo with _apt_repository_ requires the _python-apt_ package, which is installed in the first apt section.